internal/scorecard/alpha/scorecard.go

```diff
@@ -17,54 +17,74 @@ package alpha
 import (
 	"errors"
 	"fmt"
-	"io/ioutil"
-	"log"
 	"strings"
+	"time"
 
 	"github.com/operator-framework/operator-sdk/version"
-	"gopkg.in/yaml.v2"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/client-go/kubernetes"
 )
 
 type Options struct {
-	Config       Config
-	Selector     labels.Selector
-	List         bool
-	OutputFormat string
-	Client       kubernetes.Interface
+	Config          Config
+	Selector        labels.Selector
+	List            bool
+	Cleanup         bool
+	BundlePath      string
+	WaitTime        int
+	OutputFormat    string
+	Kubeconfig      string
+	Namespace       string
+	BundleConfigMap *v1.ConfigMap
+	ServiceAccount  string
+	Client          kubernetes.Interface
```

Comment:
move bools to the end to optimize the memory space. ( Is not it failing in the lint? )

Reply:
the linter didn't say anything about this, but I went ahead and moved the bools to the end, its amazing to me that golang would not be able to optimize that but oh well :^)).